lib/resources/content/javascriptservices/Views/Shared/_Layout.cshtml

```diff
@@ -4,8 +4,7 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>@ViewData["Title"] - Aurelia</title>
-
-        <link rel="stylesheet" href="~/dist/vendor.css" asp-append-version="true">
```

Comment:
Can you glob this so it won't fail if no vendor css I don't like seeing the 404 out the gate.  After that it all looks good.
<link rel="stylesheet" asp-append-version="true" asp-href-include="~/dist/vendor*.css">

Reply:
Or if you also use app*.css files.

`<link rel="stylesheet" asp-append-version="true" asp-href-include="~/dist/css/*.css"/>`